sunpy/version.py

```diff
@@ -1,13 +0,0 @@
-# This file is for compatibility with astropy_helpers
```

Comment:
We do still need this file.

Reply:
What for?

Reply:
It calculates our version at runtime: https://github.com/sunpy/sunpy/blob/master/sunpy/__init__.py#L35

Reply:
Just at runtime? The version appears in the docs build and the pypi packages that twine checks.

Reply:
Yeah, the docs call a different thing to get the version, and pypi uses setuptools_scm to generate it into the sdist metadata.